app/javascript/components/pr-feed.vue

```diff
@@ -82,6 +82,15 @@
               Dislike
             </button>
           </div>
+          <a
+            :href="'/organizations/' + organizationName + '/pull_requests/'+ pullRequest.id"
```

Comment:
Queda mejor usando interpolación de strings
```suggestion
            :href=`/organizations/${organizationName}/pull_requests/${pullRequest.id}`
```